recipe/meta.yaml

```diff
@@ -114,20 +124,25 @@ requirements:
     # Other requirements
     - cffi
     - future
-    - libuv                           # [win]
+    - libprotobuf                     # [not win]
+    # on osx, libuv supports torch.distributed support. See build.sh.
+    - libuv                           # [win or osx]
     # Pinnings as per pytorch/.circleci/docker/common/install_conda.sh,
     # except for python 3.8 because we don't have the numpy version required
-    - numpy >=1.21.2,<1.22.0 # [py>=310]
-    - numpy >=1.19.2,<1.20.0 # [py<=39]
+    - numpy >=1.21,<1.22 # [py>=310]
+    - numpy >=1.19,<1.20 # [py<=39]
```

Comment:
Will it be more clear?
    - numpy 1.21  # [py>=310]
    - numpy 1.19  # [py<=39]

Reply:
This would resolve to 1.21.0 and 1.19.0

Reply:
1.21.* and 1.19.* is also possible

Reply:
right ...

Reply:
I've left it as it is